docs/source/examples/workspace/topologies/libvirt-new.yml

```diff
@@ -7,7 +7,7 @@ resource_groups:
       - role: libvirt_node
         name: centos71
         uri: qemu:///system
-        count: 1
+        count: 2
         image_src: http://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1608.qcow2.xz
         memory: 2048
```

Comment:
Best to reduce the memory here to 1024 (unless there's a specific need for more)

Reply:
Same thing as the one above

Reply:
This change was the same as the one above.